src/markdown/markdown-body.scss

```diff
@@ -95,4 +95,12 @@
       margin-bottom: 0;
     }
   }
+
+  sup > a::before {
+    content: "[";
+  }
+
+  sup > a::after {
+    content: "]";
+  }
```

Comment:
Is there a chance that a user would want to add a link inside a `<sup>` and not get it wrapped with `[]`? It seems we currently use the `data-footnote-ref` attribute. So if we wanted to scope this a bit more, we might can instead use:

```suggestion
  [data-footnote-ref]::before {
    content: "[";
  }

  [data-footnote-ref]::after {
    content: "]";
  }
```

Not sure if `data-footnote-ref` will be part of, I think we use a "common mark" library and safe to use? 🤔

Reply:
Hmm...That's a good question. 

Yeah, `data-footnote-ref` is safe to use (I added it to the library 😄 ).

I think it's a pretty low chance that a user would want that behavior.

Reply:
Ok, we can keep it "as is" and in case we hear reports from users, switch to using `[data-footnote-ref]`.